main/document/create_audio.php

```diff
@@ -345,7 +345,7 @@ function update_voices(selectedvoicegroup){
  */
 function downloadMP3_google($filepath, $dir)
 {
-	$location='create_audio.php?'.api_get_cidreq().'&id='.Security::remove_XSS($_POST['document_id']).'&dt2a=google';
+    $location='create_audio.php?'.api_get_cidreq().'&id='.Security::remove_XSS($_POST['id']).'&dt2a=google';
```

Comment:
When you are sure a variable is an integer, please *always* use "intval()" as an integer never generates a risk of XSS.